web/src/integTest/java/com/microsoft/applicationinsights/web/spring/WebTelemetryTests.java

```diff
@@ -21,263 +21,207 @@
 
 package com.microsoft.applicationinsights.web.spring;
 
-import com.microsoft.azure.storage.CloudStorageAccount;
-import com.microsoft.azure.storage.blob.CloudBlobClient;
-import com.microsoft.azure.storage.blob.CloudBlockBlob;
-import com.microsoft.azure.storage.queue.CloudQueue;
-import com.microsoft.azure.storage.queue.CloudQueueClient;
-import com.microsoft.azure.storage.queue.CloudQueueMessage;
-import org.apache.commons.lang3.time.StopWatch;
-import org.json.JSONException;
-import org.json.JSONObject;
+import com.microsoft.applicationinsights.framework.ApplicationTelemetryManager;
+import com.microsoft.applicationinsights.framework.HttpRequestClient;
+import com.microsoft.applicationinsights.framework.TestEnvironment;
+import com.microsoft.applicationinsights.framework.UriWithExpectedResult;
+import com.microsoft.applicationinsights.framework.telemetries.RequestTelemetryItem;
+import com.microsoft.applicationinsights.framework.telemetries.TelemetryItem;
+import com.microsoft.applicationinsights.internal.util.LocalStringsUtils;
+import com.microsoft.applicationinsights.web.utils.HttpHelper;
+import org.eclipse.jetty.http.HttpStatus;
+import org.junit.Assert;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
-import java.net.HttpURLConnection;
 import java.net.URI;
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.concurrent.TimeoutException;
+import java.text.ParseException;
+import java.util.*;
 
 /**
  * Created by moralt on 05/05/2015.
  */
 public class WebTelemetryTests {
-    String runId = Helpers.getRandomUUIDString();
-    TestEnvironment testEnv;
-    TestSettings testSettings;
-    private static final int millisecondsInSecond = 1000;
 
-    /**
-     * Sends GET requests to server and expects that will telemetry from app insights and it will include the correct information about the request
-     * @throws Exception
-     */
-    @Test
-    public void sendHttpRequestsTest() throws Exception {
+    // region Tests run IDs
 
-        initTestConfiguration();
+    private static final String TEST_REQUEST_COMMON_PROPERTIES_RUN_ID = "1";
```

Comment:
Please use generated GUIDs rather than static numbers.
Reason - if we'll want to run tests against Tomcat and Jetty (or just different app server versions) in parallel then using a static ID would cause a clash